zmsapi/routing.php

```diff
@@ -1899,7 +1899,7 @@
  *                                  $ref: "schema/log.json"
  */
 \App::$slim->get(
-    '/log/process/{id:\d{1,11}}/',
+    '/log/process/{id}/',
```

Comment:
Die Swagger-Dokumentation ist nicht konsistent mit der neuen Änderung, da der id-Parameter noch als type: **number** definiert ist, obwohl er anscheinend jetzt ein String ist.
Änderungsvorschlag: Aktualisiere den Typ des id-Parameters in der Swagger-Doku auf type: **string**:

Bsp.:
```
/**
 *  @swagger
 *  "/log/process/{filter}/":
 *      get:
 *          summary: Get a list of log entries for a process
 *          x-since: 2.11
 *          tags:
 *              - process
 *              - log
 *          parameters:
 *              -   name: X-Authkey
 *                  required: true
 *                  description: authentication key to identify user for testing access rights
 *                  in: header
 *                  type: string
 *              -   name: filter
 *                  description: Filter criteria to search logs (e.g., part of ID, name, or other object data)
 *                  in: path
 *                  required: true
 *                  type: string
 *          responses:
 *              200:
 *                  description: Get a list of log entries for a process or empty list
 *                  schema:
 *                      type: object
 *                      properties:
 *                          meta:
 *                              $ref: "schema/metaresult.json"
 *                          data:
 *                              type: array
 *                              items:
 *                                  $ref: "schema/log.json"
 */
 ```
Es ist unklar, wie viele Unit Tests durch die Anpassungen im `schema/log.json` beeinflusst werden.